File: umpire.cs
using System;

public class Program
{
	public static void Main()
	{
		int n = int.Parse(Console.ReadLine());
		int strike = 3;
		int ball = 4;
		while (n-- > 0)
		{
			string s = Console.ReadLine();
			if ("strike".Equals(s)) Console.WriteLine(--strike > 0 ? "strike!" : "out!");
			else if ("ball".Equals(s)) Console.WriteLine(--ball > 0 ? "ball!" : "fourball!");
		}
	}
}

File: umpire.swift
let n = Int(readLine()!)!
var strike = 3
var ball = 4
for _ in 0..<n {
	let s = readLine()!
	if s == "strike" {
		strike -= 1
		print(strike > 0 ? "strike!" : "out!")
	} else if s == "ball" {
		ball -= 1
		print(ball > 0 ? "ball!" : "fourball!")
	}
}
